components/landing/TheQuickStart/ProvidersList.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <section class="providers-list">
+    <h4>Providers</h4>
+    <ul class="providers-list__list">
+      <template v-for="(item, index) in providersList">
+        <li
+          v-if="item.docsCta && item.docsCta.url"
```

Comment:
I believe this could be simplified with [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining):

```suggestion
          v-if="item.docsCta?.url"
```

Reply:
This breaks for me, for some reason:


https://user-images.githubusercontent.com/6276074/226467513-b5c481ff-ec53-4330-afb8-866d5901dbdb.mov

Reply:
I just tested it locally and get the same results as you. This is interesting, I wonder why this is... 🤔 Maybe it's related to the compiler having some outdated dependency.

In any case, the code as you wrote it is fine. There is no need to change it.